spring-cloud-deployer-cloudfoundry/pom.xml

```diff
@@ -43,6 +43,12 @@
 			<groupId>org.cloudfoundry</groupId>
 			<artifactId>cloudfoundry-client-reactor</artifactId>
 			<version>${cloudfoundry-java-lib.version}</version>
+			<exclusions>
```

Comment:
@corneil do you mind adding a well-known comment in each of these exclusions so we can easily find it later. 

How about `// @TODO boot3 jakarta` ?

Reply:
Updated